aodndata/aatams_sattag_qc_dm/aatams_sattag_qc_dm.py

```diff
@@ -0,0 +1,40 @@
+"""handler for the AATAMS_SATTAG_QC_DM handler."""
+import os
+
+from aodncore.pipeline import HandlerBase, PipelineFilePublishType
+from .aatams_sattag_qc_dm_schema import AATAMS_SATTAG_QC_DM_SCHEMA
```

Comment:
Yeap, I overstep pep8 to make this strange on purpose! The rationale was to avoid name mixing. Note this is not a handler but a "constant" wrapper class to execute the validation inside the handler. Because the class is constant and state do not change, I thought it was a good idea to use a constant namespace (UPPERCASE).

I will rename to CamelCase.